pom.xml

```diff
@@ -20,18 +20,18 @@
   <parent>
     <groupId>io.openshift</groupId>
     <artifactId>booster-parent</artifactId>
-    <version>21</version>
+    <version>22</version>
```

Comment:
This seems to have updated some Arquillian version. Do we need to do the same or do we just reuse the deps from the parent? I am assuming it's the latter, but just wanted to double check with you

Reply:
Yes, I'm worried about the implications as well. Do we have a test run to support that change?

Reply:
I'm believe running any of the booster tests would do the job

Reply:
Just ran IT on HTTP and CB boosters and all works fine

Reply:
Health check works fine too